scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -119,6 +118,8 @@
 
     private CheckEligibleTaskDescriptorScript checkEligibleTaskDescriptorScript;
 
+    private SchedulingMainLoopTimingLogger schedulingMainLoopTimingLogger;
```

Comment:
could you change the following line in this class:

public static final Logger logger = Logger.getLogger(SchedulingService.class);

to

public static final Logger logger = Logger.getLogger(SchedulingMethodImpl.class)